src/generation.rs

```diff
@@ -1356,11 +1356,6 @@ impl GenerationScope {
             // new
             for variant in variants.iter() {
                 let variant_arg = variant.name_as_var();
-                let enc_fields = if CLI_ARGS.preserve_encodings {
```

Comment:
was this just unused before?

Reply:
It was not needed. It might have been used before we changed how enums handle storing/using encoding variables as those now have a dedicated struct to abstract all of that away starting in #52.